common.sh

```diff
@@ -28,6 +28,13 @@ fi
 source $CONFIG
 cat $CONFIG
 
+# Connect to system libvirt
+export LIBVIRT_DEFAULT_URI=qemu:///system
```

Comment:
One problem is this isn't set by default in the environment, so you still need to either set it, or use sudo to see the VMs via virsh list outside of the scripts.  Can we perhaps set this globally somewhere?